src/xo-app/pool/tab-logs.js

```diff
@@ -1,111 +1,59 @@
 import _ from 'intl'
-import ActionRow from 'action-row-button'
 import React, { Component } from 'react'
-import TabButton from 'tab-button'
+import SortedTable from 'sorted-table'
 import { deleteMessage } from 'xo'
-import { createPager, createSelector } from 'selectors'
 import { FormattedRelative, FormattedTime } from 'react-intl'
-import { Container, Row, Col } from 'grid'
 import {
-  ceil,
-  isEmpty,
   map
 } from 'lodash'
 
-const LOGS_PER_PAGE = 10
-
-export default class TabLogs extends Component {
-  constructor () {
-    super()
-
-    this.getLogs = createPager(
-      () => this.props.logs,
-      () => this.state.page,
-      LOGS_PER_PAGE
-    )
-
-    this.getNPages = createSelector(
-      () => this.props.logs ? this.props.logs.length : 0,
-      nLogs => ceil(nLogs / LOGS_PER_PAGE)
-    )
+const COLUMNS = [
+  {
+    itemRenderer: log => <div>
+      <FormattedTime value={log.time * 1000} minute='numeric' hour='numeric' day='numeric' month='long' year='numeric' /> (<FormattedRelative value={log.time * 1000} />)
+    </div>,
+    name: _('logDate'),
+    default: true,
+    sortCriteria: 'time'
+  },
+  {
+    itemRenderer: log => log.name,
+    name: _('logName'),
+    sortCriteria: 'name'
+  },
+  {
+    itemRenderer: log => log.body,
+    name: _('logContent'),
+    sortCriteria: 'body'
+  }
+]
 
-    this.state = {
-      page: 1
-    }
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: deleteMessage,
+    icon: 'delete',
+    label: 'deleteLog'
```

Comment:
Shouldn't the `label` be an actual node and not just a message key? (same question for the grouped actions)